operational/azure/aks_nodepools_without_autoscaling/aks_nodepools_without_autoscaling.pt

```diff
@@ -0,0 +1,217 @@
+name "AKS Node Pools Without Autoscaling"
+rs_pt_ver 20180301
+type "policy"
+short_description "Raise an incident if there are any AKS user node pools without autoscaling enabled. \n See the [README](https://github.com/flexera/policy_templates/tree/master/operational/azure/aks_nodepools_without_autoscaling) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
+long_description ""
+category "Operational"
+severity "low"
+default_frequency "daily"
+info(
+  version: "2.0",
+  provider: "Azure",
+  service: "AKS",
+  policy_set: ""
+)
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses of the recipients you wish to notify."
+end
+
+parameter "param_exclusion_tags" do
+  category "User Inputs"
+  label "Tags To Ignore"
+  description "The policy will ignore any clusters with the specified tags. Only supply the tag keys; the policy assumes that the tag value is irrelevant."
+  type "list"
+  default []
```

Comment:
Can you remove this?

Reply:
@rshade The entire thing or just the "default []"? If the former, I'm happy to do so, but I'd be curious why since my understanding is that we generally want to give the option to filter for tags.